frontend/src/Settings/index.jsx

```diff
@@ -1,14 +1,105 @@
-import React from 'react';
-import { View, Text } from 'react-native';
-import styles from '../common/styles';
+import React, { useState } from 'react';
+import { useSelector } from 'react-redux';
+import {
+  View, Text, Alert, StyleSheet, ImageBackground,
+} from 'react-native';
+import Colors from '../common/colors';
+
+import background from '../../assets/login/background.jpg';
+
+import InputField from './InputField';
+import SubmitButton from './SubmitButton';
+
+const styles = StyleSheet.create({
+  titleText: {
+    margin: 20,
+    fontSize: 36,
+    color: Colors.whiteColor,
+    fontWeight: 'bold',
+  },
+  labelText: {
+    width: 240,
+    fontSize: 16,
+    color: Colors.whiteColor,
+  },
+  buttonRow: {
+    width: 240,
+    height: 60,
+    paddingTop: 6,
+    flexDirection: 'row-reverse',
+    alignItems: 'flex-start',
+  },
+  container: {
+    paddingTop: 40,
+    flexDirection: 'column',
+    alignItems: 'center',
+  },
+  background: {
+    width: '100%',
+    height: '100%',
+  },
+});
 
 function Settings() {
+  const [emailInput, setEmailInput] = useState('');
+  const [usernameInput, setUsernameInput] = useState('');
+  const [kingdomNameInput, setKingdomNameInput] = useState('');
+  const [passwordInput, setPasswordInput] = useState('');
+
+  function handleSubmit() {
+    const reqObj = {};
```

Comment:
`reqObj` is not a descriptive name, please rename it to something more meaningful! (`settings` is more descriptive, or even `user`, `user.email, user.kingdomname`..., what do you think?)